src/mini_s3.erl

```diff
@@ -326,7 +326,7 @@ s3_url(Method, BucketName0, Key0, Lifetime, RawHeaders, Date, Config)
 %                     Lifetime
 %
 % given a TTL, x-amz-expires should be set to be the closest expiry-window
-% boundary >= present + TTL.
+% boundary >= present + TTL, ie present+TTL selects the expiry-window.
```

Comment:
I am not sure this comment adds much.

Reply:
That comment was very helpful to explain to myself what I had done when I had to "re- figure out" what I had done :)  Do you mind if we keep it, for my own future sanity?